apps/chat-e2e/src/ui/pages/dialHomePage.ts

```diff
@@ -39,4 +43,17 @@ export class DialHomePage extends BasePage {
         .waitForState({ state: 'attached' });
     }
   }
+
+  async reloadPage() {
+    await super.reloadPage();
+    const appContainer = this.getAppContainer();
+    await appContainer
+      .getChatBar()
+      .getChatLoader()
+      .waitForState({ state: 'hidden' });
+    await appContainer
```

Comment:
Is it correct that you are doing this twice?

Reply:
this is done for both side panels - chat and promts:

`await appContainer
      .getChatBar()
      .getChatLoader()
      .waitForState({ state: 'hidden' });

    await appContainer
      .getPromptBar()
      .getChatLoader()
      .waitForState({ state: 'hidden' });`